src/Commands/Create.php

```diff
@@ -21,7 +21,11 @@ public function handle()
             ?: config('db-snapshots.default_connection')
             ?? config('database.default');
 
-        $snapshotName = $this->argument('name') ?? Carbon::now()->format('Y-m-d_H-i-s');
+        if (! is_null($this->option('connection')) && is_null($this->argument('name'))) {
```

Comment:
Could you refactor this to avoid using. `else`. Maybe put this logic in a dedicated function with early returns.

Reply:
Extracted to a private method